adal/build.gradle

```diff
@@ -78,6 +79,40 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+
+task createPom  {
+    pom {
+        project {
+            groupId 'com.microsoft.aad'
+            artifactId 'adal-aggregator'
+            version '1.0.0-SNAPSHOT'
+            packaging 'jar'
+            name 'ADAL ANDROID-API'
```

Comment:
>name 'ADAL ANDROID-API' [](start = 12, length = 23)

can we check this pom.xml and make sure everything is correct? [https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/v1.2.2/src/pom.xml](https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/v1.2.2/src/pom.xml)